core/templates/services/site-analytics.service.ts

```diff
@@ -20,35 +20,34 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
-import constants from 'assets/constants';
 import { WindowRef } from 'services/contextual/window-ref.service';
+import { initializeGoogleAnalytics } from 'google-analytics.initializer';
 
 // Service for sending events to Google Analytics.
-//
-// Note that events are only sent if the CAN_SEND_ANALYTICS_EVENTS flag is
-// turned on. This flag must be turned on explicitly by the application
-// owner in feconf.py.
 
 @Injectable({
   providedIn: 'root'
 })
 export class SiteAnalyticsService {
-  constructor(private windowRef: WindowRef) {}
+  static googleAnalyticsAreInitialized: boolean = false;
```

Comment:
Probably `googleAnalyticsIsInitialized` since Google Analytics is really one thing. Ditto elsewhere.

Reply:
done